Jenkinsfile

```diff
@@ -1,5 +1,7 @@
-buildPlugin(useContainerAgent: false, configurations: [
-  [ platform: "linux", jdk: "11" ],
-  [ platform: "windows", jdk: "11" ],
-  [ platform: "linux", jdk: "17" ]
+buildPlugin(
+  useContainerAgent: true,
+  configurations: [
+    [platform: "linux", jdk: "11"],
+    [platform: 'linux', jdk: 21],
+    [platform: 'windows', jdk: 17],
```

Comment:
Probably no need of 17.
Per my understanding Jenkins will jump directly from 11 to 21.
So 17 seems to be a no need for testing?
I would rather prefer having windows 11 and/or 21.

Reply:
Starting soon, the weekly Jenkins releases will include an administrative monitor to notify users that Jenkins will no longer support Java 11 after September 30, 2024.

The official documentation is currently being updated to support JDK17. Additionally, starting this week, the default Docker images will use JDK17. JDK17 is now the preferred version for Jenkins until we eventually transition to JDK21.

For testing purposes, it's important to check the compatibility of your plugin with both JDK17 and JDK21. Regarding JDK11, the choice to continue using it is up to you. 🤗